### PR TITLE
transmission: build with qt6

### PR DESCRIPTION
**Pull request by dkwo**
- I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

**Comment by dkwo** [comment]
for cross:
```
[287/379] Automatic MOC and UIC for target transmission-qt
[288/379] Generating transmission_af.qm
FAILED: qt/transmission_af.qm /builddir/transmission-4.0.5/build/qt/transmission_af.qm 
cd /builddir/transmission-4.0.5/build/qt && /usr/aarch64-linux-gnu/usr/lib64/qt6/bin/lrelease -silent /builddir/transmission-4.0.5/qt/translations/transmission_af.ts -qm /builddir/transmission-4.0.5/build/qt/transmission_af.qm
/bin/sh: line 1: /usr/aarch64-linux-gnu/usr/lib64/qt6/bin/lrelease: cannot execute binary file: Exec format error
```

File: srcpkgs/transmission/template [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+ -DENABLE_CLI=ON
+ -DENABLE_GTK=ON
+ -DENABLE_QT=ON
+ -DUSE_QT_VERSION=6
+ -DWITH_SYSTEMD=OFF
+ -DENABLE_MAC=OFF"
+hostmakedepends="intltool glib-devel libtool pkg-config qt6-base qt6-tools"
 makedepends="dbus-glib-devel gtkmm4-devel glibmm-devel
- libcurl-devel libevent-devel qt5-tools-devel qt5-svg-devel
+ libcurl-devel libevent-devel qt6-tools-devel qt6-svg-devel
```
Comment (chrysos349) [inline srcpkgs/transmission/template:15]: remove `qt6-tools-devel` to fix cross.

Comment (dkwo) [reply srcpkgs/transmission/template:15]: thanks, that worked.

File: srcpkgs/transmission/template [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
- libtool pkg-config qt5-host-tools qt5-qmake"
+configure_args="
+ -DENABLE_CLI=ON
+ -DENABLE_GTK=ON
+ -DENABLE_QT=ON
+ -DUSE_QT_VERSION=6
+ -DWITH_SYSTEMD=OFF
+ -DENABLE_MAC=OFF"
+hostmakedepends="intltool glib-devel libtool pkg-config qt6-base
+ qt6-tools-devel"
```
Comment (sgn) [inline srcpkgs/transmission/template:14]:
```suggestion
 qt6-tools"
```

Comment (dkwo) [reply srcpkgs/transmission/template:14]: done, thanks.

**Pull request merged**